tests/integration/targets/lambda/tasks/main.yml

```diff
@@ -8,6 +8,21 @@
   collections:
     - community.general
   block:
+  - name: create minimal lambda role
+    iam_role:
+      name: '{{ lambda_role_name }}'
+      assume_role_policy_document: '{{ lookup("file", "minimal_trust_policy.json") }}'
+      create_instance_profile: false
+      managed_policies:
+      - 'arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole'
```

Comment:
```suggestion
```
This policy is not strictly necessary for these tests to run.  Digging into our `ansible_lambda_role`, it only has AllowXRayPutTraceSegments.  With AWSLambdaBasicExecutionRole enabled, lambda events are uploaded to CloudWatch with a log group that doesn't match any of our existing permissions for CloudWatch in the terminator that look like: `log-group:/aws/lambda/shippable-196-9:log-stream`.  We're having to clean them up manually after this PR runs.  Do we have reason to want to create these logs?  If not we should remove this policy, otherwise we need to add an appropriate resource in the terminator CloudWatch policies.